romancal/tweakreg/tweakreg_step.py

```diff
@@ -223,7 +223,16 @@ def process(self, input):
                 mask = (x > xmin) & (x < xmax) & (y > ymin) & (y < ymax)
                 catalog = catalog[mask]
 
-            filename = image_model.meta.filename
+            n_removed_src = np.sum(np.logical_not(mask))
+            if n_removed_src:
+                self.log.info(
+                    f"Removed {n_removed_src} sources from {filename}'s "
+                    "catalog that were outside of the bounding box."
+                )
```

Comment:
The variable `mask` was defined inside an `if` statement block and referenced outside of it. Maybe we should move these lines (226-231) into the `if` block?